### ISSUE TITLE
cdktf get failed with scalaway provider version 2.3.0

### ISSUE THREAD

**Issue by jbeduneauLEGALY** [scaleway/terraform-provider-scaleway#1482]
When I launch cdktf get command since last version, I get this error
  | providers/scaleway/object.ts:171:35 - error TS2339: Property 'keys' does not exist on type 'typeof Object'.
  | 171       this.isEmptyObject = Object.keys(value).length === 0;

Please find the complete log further:

```
NOTE: Temp directory retained due to an error: /var/folders/ds/pz3w4qqj4rvcx0y37kr9z0dc0000gn/T/temp-2REOgV
Error: jsii compilation failed with non-zero exit code: 1
  | [2022-09-23T06:48:54.013] [ERROR] jsii/compiler - Compilation errors prevented the JSII assembly from being created
  | warning JSII6: A "peerDependency" on "constructs" at "10.1.92" means you should take a "devDependency" on "constructs" at "10.1.92" (found "undefined")
  | warning JSII6: A "peerDependency" on "cdktf" at "0.12.2" means you should take a "devDependency" on "cdktf" at "0.12.2" (found "undefined")
  | warning JSII3: There is no "README.md" file. It is required in order to generate valid PyPI (Python) packages.
  | providers/scaleway/object.ts:171:35 - error TS2339: Property 'keys' does not exist on type 'typeof Object'.
  | 171       this.isEmptyObject = Object.keys(value).length === 0;
  |                                       ~~~~
  +----------------------------------------------------------------------------------
  | Command: /usr/local/lib/node_modules/cdktf-cli/node_modules/jsii/bin/jsii --silence-warnings reserved-word
  | Workdir: /var/folders/ds/pz3w4qqj4rvcx0y37kr9z0dc0000gn/T/temp-2REOgV
  +----------------------------------------------------------------------------------
    at p (/usr/local/lib/node_modules/cdktf-cli/bundle/bin/cmds/handlers.js:52:1581)
    at ChildProcess.<anonymous> (/usr/local/lib/node_modules/cdktf-cli/bundle/bin/cmds/handlers.js:55:112)
    at Object.onceWrapper (events.js:483:26)
    at ChildProcess.emit (events.js:376:20)
    at ChildProcess.emit (domain.js:470:12)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:277:12)
```

Thanks by advance

**Comment by Codelax** [comment]
Hi, thanks for reporting. I created an [issue](https://github.com/hashicorp/terraform-cdk/issues/2137) on cdktf repository.

**Comment by Codelax** [comment]
Should be fixed in cdktf 0.13.1
